### ISSUE TITLE
Error initiating app on non-interactive mode

### ISSUE THREAD

**Issue by thiagobrez** [expo/expo-cli#1141]
## Description

Hi!

I'm filling this issue here because I first created at the forums (https://forums.expo.io/t/missing-docs-on-non-interactive-mode/28244) and got no response after 20 days.

I’m creating a CLI tool and want to run expo init in the background, therefore using non-interactive mode. But I can’t find a way to start with the blank (TypeScript) template.

## Expected Behavior

To start a new app with blank typescript template using non-interactive mode.

## Observed Behavior

The closest I’ve got was:

`expo init my-app --non-interactive --template blank --name MyApp --android-package com.mycompany.myapp --ios-bundle-identifier com.mycompany.myapp`

But this fails with error:

```
[13:58:17] Can't read JSON file: /Users/MyUser/Repos/my-app/app.json
└─ Cause: Error: ENOENT: no such file or directory, open '/Users/MyUser/Repos/my-app/app.json'
[13:58:17] JsonFileError: Can't read JSON file: /Users/MyUser/Repos/my-app/app.json
└─ Cause: Error: ENOENT: no such file or directory, open '/Users/MyUser/Repos/my-app/app.json'
    at /Users/MyUser/.config/yarn/global/node_modules/@expo/json-file/src/JsonFile.ts:158:13
    at Generator.throw (<anonymous>)
    at rejected (/Users/MyUser/.config/yarn/global/node_modules/@expo/json-file/build/JsonFile.js:5:65)
```

And also, if this worked, probably wouldn’t start using the TypeScript template.

Any help would be appreciated,
Thanks!

**Comment by fson** [comment]
Looks like this is a duplicate of #1133, which will likely be fixed by #1137.

**Comment by thiagobrez** [comment]
@fson Oops, sorry about that. But what about starting with blank TypeScript template?
